Bottom: 91f03b2c47b6021aa8cbabcb4da4f7e9e3eab4a7
Top:    6d4809410bf47e8668d142a8d06c52153204c9d8
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2009-10-21 21:28:08 +0100

Not sure what this patch is for ;)


---

diff --git a/src/draw.c b/src/draw.c
index 59e5e31..6089eba 100644
--- a/src/draw.c
+++ b/src/draw.c
@@ -530,7 +530,7 @@ DrawEverything (BoxTypePtr drawn_area)
 	r_search (PCB->Data->element_tree, drawn_area, NULL, EMark_callback,
 		  NULL);
       /* Draw rat lines on top */
-      if (PCB->RatOn)
+      if (PCB->RatOn && gui->set_layer ("rats", SL (RATS, 0), 0))
 	DrawRats(drawn_area);
     }
